blueocean-bitbucket-pipeline/src/main/java/io/jenkins/blueocean/blueocean_bitbucket_pipeline/BitbucketPipelineCreateRequest.java

```diff
@@ -68,6 +69,7 @@ protected SCMSource createSource(@Nonnull MultiBranchProject project, @Nonnull B
                 .withTrait(new OriginPullRequestDiscoveryTrait(strategies))
                 .withTrait(new CleanBeforeCheckoutTrait())
                 .withTrait(new CleanAfterCheckoutTrait())
```

Comment:
Why clean after if you clean before? (clean before is needed in case the workspace touched some files an blocks the checkout)